main.css

```diff
@@ -7,6 +7,79 @@ html, body
     flex-direction: column;
 }
 
+footer
+{
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    width: 100%;
+    height: 20px;
+    z-index: 9999;
+    background-color: transparent;
+}
+
+#notifications
+{
+    position: fixed;
+    top: 45px;
+    right: 0px;
+    height: auto;
+    width: 25%;
+    margin-top: 5px;
```

Comment:
Let's just make this `margin: 5px`, because we want clearing on the left, right, top, and down anyway